source/refunding_payments/index.html.md.erb

```diff
@@ -182,70 +180,71 @@ For example, the response body for a refund would include:
 ...
 ```
 
-Initially, in a live environment, the status returned will be `submitted`.
+In a live environment, the status returned will initially be `submitted`.
 After the PSP has processed the refund, the status returned will be
-`success` or `error`. (In the test environment, the status will always
-go straight to `success`).
+`success` or `error`. In a test environment, the status will always
+go directly to `success`.
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
 | Refund processing status | Meaning                                                                                                            |
 |--------------------------|--------------------------------------------------------------------------------------------------------------------|
-| submitted                | The refund request is valid as far as GOV.UK Pay can tell and has been passed to the underlying payment processor. |
+| submitted                | The refund request is valid as far as GOV.UK Pay can determine. It has been passed to the underlying payment service provider. |
 | success                  | The refund has been successfully processed.                                                                        |
-| error                    | It was not possible for the payment processor to make the refund.                                                  |
+| error                    | It was not possible for the payment service provider to make the refund.                                                  |
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-To manage this, you must use <a
+To manage refunds in live environments, you must use <a
 href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/payment-id/find-payment-refund-by-id"
 target="blank">Find payment refund by ID</a> (link opens in new window) to
-check the processing status of the refund until it changes to either
-``success`` or ``error``.
+check the processing status of the refund, until it changes to either
+`success` or `error`.
 
 It will typically take 30 minutes for the status to change. You should not
 check the status more than once every 5 minutes.
 
 In the event of an error, GOV.UK Pay will not currently provide any more
 information. Please [contact
-us](/support_contact_and_more_information/#contact-us) if more information is
-required about why a refund failed.
+us](/support_contact_and_more_information/#contact-us) if you need more
+information required about why a refund failed.
 
-## Refunding from the GOV.UK Pay admin site
+## Refunding from the GOV.UK Pay admin tool
```

Comment:
can i doublecheck whether 'admin tool' is the preferred moniker throughout the rest of docs?

Reply:
It should be; I've had to try to clean up a lot of things like this as I've been going along but it's one of the things I've made a note to go back and check throughout the whole documentation before I finish on Pay